### PR TITLE
End Round button

### PR DESCRIPTION
**Pull request by quardbreak**
Adds the round end button. The difference between reboot - it makes game end smoothly with credits and round result print.
Formatted some button names and removed admin name duplication for some logs.
Adds coordinate output log for the Create Object panel, moves the spawn button to right from the Number of matched field: 
image

### CODE REVIEW

File: code/modules/admin/admin.dm [hunk 1]
```diff
@@ -848,6 +848,24 @@ var/global/floorIsLava = 0
...
 
+/datum/admins/proc/endnow()
+	set category = "Server"
+	set desc = "Ending game round"
+	set name = "End Round"
+	if(!usr.client.holder || !check_rights(R_ADMIN))
+		return
+
+	if(GAME_STATE != RUNLEVEL_GAME)
+		to_chat(usr, "<span class='bigdanger'>You can use this only with the active round.</span>")
```
Comment (MistakeNot4892) [inline code/modules/admin/admin.dm]: Suggest swapping this to `The round has not started yet!`.

File: code/modules/admin/admin.dm [hunk 2]
```diff
@@ -907,12 +925,12 @@ var/global/floorIsLava = 0
...
 
 	config.ninjas_allowed = !config.ninjas_allowed
-	log_and_message_admins("toggled Space Ninjas [config.ninjas_allowed ? "on" : "off"].")
+	log_and_message_admins("toggled space ninjas [config.ninjas_allowed ? "on" : "off"].")
 	SSstatistics.add_field_details("admin_verb","TSN") //If you are copy-pasting this, ensure the 2nd parameter is unique to the new proc!
 
 /datum/admins/proc/delay()
 	set category = "Server"
-	set desc="Delay the game start/end"
+	set desc="Delay the Game start/end"
```
Comment (MistakeNot4892) [inline code/modules/admin/admin.dm]: Why make this uppercase?

File: code/modules/admin/admin.dm [hunk 3]
```diff
@@ -931,23 +949,23 @@ var/global/floorIsLava = 0
 
 /datum/admins/proc/adjump()
 	set category = "Server"
-	set desc="Toggle admin jumping"
+	set desc="Toggle Admin Jumping"
```
Comment (MistakeNot4892) [inline code/modules/admin/admin.dm]: I don't understand why you made the ones above lowercase and then made these ones uppercase.

**Review by MistakeNot4892 — APPROVED** [review]

File: code/modules/admin/topic.dm [hunk 4]
```diff
@@ -1713,7 +1713,7 @@
 									var/mob/M = O
 									M.real_name = obj_name
 
-		log_and_message_admins("created [number] [english_list(paths)]")
+		log_and_message_admins("created [number] [english_list(paths)] at ([target.x],[target.y],[target.z])")
```
Comment (PsiOmegaDelta) [inline code/modules/admin/topic.dm:1716]: `log_and_message_admins()` already print X/Y/Z information if supplied a valid target

**Pull request merged**